index.js

```diff
@@ -126,7 +126,9 @@ function replacePath (path, keys) {
       name: key || index++,
       delimiter: prefix || '/',
       optional: optional,
-      repeat: repeat
+      repeat: repeat,
+      offset: i + (prefix ? prefix.length : 0),
```

Comment:
Yes, I know. You're right. The omit there is actually because I was too lazy to update all the tests immediately, I'll be removing that later. I don't think there's anything I actually need `lodash` for, but you're right that build size is a valid concern (not that it matters in tests, like you mentioned).